tests/INP-cylinder4

```diff
@@ -0,0 +1,7 @@
+Test file for right cylinder macrobody facet
+4 0 1.2 -2
```

Comment:
Interesting.... I do not know if it is valid MCNP syntax to use the facet of a macrobody that is not otherwise used????